pkg/vpcmodel/grouping.go

```diff
@@ -15,7 +15,7 @@ const commaSeparator = ","
 type groupingConnections map[EndpointElem]map[string]*groupedNodesInfo
 
 type groupedNodesInfo struct {
```

Comment:
Should the struct be renamed to `groupedExternalNodesInfo`?

Reply:
right, updated